collections/_article/maintenance-release-godot-4-0-3.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Maintenance release: Godot 4.0.3"
+excerpt: "A fresh portion of stability improvements and bugfixes for Godot 4.0 projects, 4.0.3 covers almost every area of the engine. It also comes with a massive bump in documentation coverage thanks to the newly updated rendering docs!"
+categories: ["release"]
+author: Yuri Sizov
+image: /storage/blog/covers/maintenance-release-godot-4-0-3.jpg
+image_caption_title: Godot Juice
+image_caption_description: A course about making a juicy game in Godot 4 by MrEliptik
```

Comment:
```suggestion
image_caption_description: A Godot 4 course by MrEliptik
```

I usually try to keep this relatively short ("A game by <author>") to avoid wrapping, leaving the details to the credit blurb.